### PR TITLE
Compass UI update

### PR DESCRIPTION
**Pull request by iguannalin**
## Description

Updated jQuery/axios functions to alpine/fetch, maximized screen space for ease of use; removed `static/` directory, as it only had `compass.js` whose functions have now been moved to inside of `compass.html `

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: templates/compass.html [hunk 1]
```diff
@@ -9,119 +9,160 @@ <h2>Compass<sub class="is-size-7 pl-1">find your way</sub></h2>
...
-                        {% for adv in adversaries %}
-                        <option value="{{ adv.adversary_id }}">{{ adv.name }}</option>
-                        {% endfor %}}
-                    </select>
+        <div class="is-flex is-flex-direction-row mb-4">
+            <div class="is-flex is-flex-direction-column">
+                <label for="layer-selection-adversary" class="pb-2 is-size-6">Generate Layer</label>
+                <div class="is-flex is-flex-direction-row is-align-items-center">
+                    <div id="layerSelectionAdversary" class="mt-1 mr-2">
+                        <select id="layer-selection-adversary" x-model="selectedAdversaryID">
```
Comment (argaudreau) [inline templates/compass.html]: The select dropdown isn't styled like the other dropdowns across the site. I think if you add "select" as a class to the div on line 16 it should fix it

File: templates/compass.html [hunk 2]
```diff
@@ -9,119 +9,160 @@ <h2>Compass<sub class="is-size-7 pl-1">find your way</sub></h2>
...
+                            <option value="" selected>Select an Adversary (All)</option>
+                            {% for adv in adversaries %}
+                            <option value="{{ adv.adversary_id }}">{{ adv.name }}</option>
+                            {% endfor %}}
+                        </select>
+                    </div>
+                    <label for="generateLayer">
+                        <button id="generateLayer" class="button is-primary is-small"
```
Comment (argaudreau) [inline templates/compass.html:29]: We could use a Bulma style here that makes it clear that the button and the select input go together: 

https://bulma.io/documentation/form/general/#form-addons (the first example with the blue search button)

File: templates/compass.html [hunk 3]
```diff
@@ -9,119 +9,160 @@ <h2>Compass<sub class="is-size-7 pl-1">find your way</sub></h2>
...
-            <div style="flex:37%;text-align:left;padding:15px;">
-                <h2>Generate Adversary</h2>
-                <button id="generateAdversary" type="button" class="button-success"
-                        style="margin-top:-3px" onclick="uploadAdversaryLayerButtonFileUpload()">Upload
-                    Adversary
-                    Layer
-                </button>
-                <input id='adversaryLayerInput' type='file' hidden/>
+            <div class="is-flex is-flex-direction-column ml-6">
+                <label for="generateAdversary" class="pb-2 is-size-6">Generate Adversary</label>
```
Comment (argaudreau) [inline templates/compass.html]: This is kind of silly, but for some reason clicking on the label text also opens the dialog to upload a file (at least in FF). Not a big deal but could be an accessibility issue down the road

File: templates/compass.html [hunk 4]
```diff
@@ -9,119 +9,160 @@ <h2>Compass<sub class="is-size-7 pl-1">find your way</sub></h2>
...
+                            {% for adv in adversaries %}
+                            <option value="{{ adv.adversary_id }}">{{ adv.name }}</option>
+                            {% endfor %}}
+                        </select>
+                    </div>
+                    <label for="generateLayer">
+                        <button id="generateLayer" class="button is-primary is-small"
+                                x-on:click="generateLayer()"><i class="pr-1 fas fa-download"></i>
+                            <span class="has-tooltip-multiline has-tooltip-bottom"
+                                  data-tooltip="Click 'Generate Layer'. Click '+' to open a new tab in the navigator. Select 'Open Existing Layer'. Select 'Upload from local' and upload the generated layer file.">Generate Layer
```
Comment (argaudreau) [inline templates/compass.html]: Is there anyway to format this with newlines in the tooltip? like:
> 1. Click 'Generate Layer'...
> 2. Click '+'....
> 3. etc.

**Review by argaudreau — CHANGES_REQUESTED** [review]
Overall looks great! Let's see how long we can get away with keeping the slogan in there 🤣

File: templates/compass.html (hunk 3 above)
Comment (iguannalin) [reply templates/compass.html]: Great catch!

File: templates/compass.html (hunk 4 above)
Comment (iguannalin) [reply templates/compass.html]: Unfortunately not, I did try to pare down the tooltip messaging more so they're not as chunky.

File: templates/compass.html [hunk 5]
```diff
@@ -1,127 +1,174 @@
...
-                        <option value="{{ adv.adversary_id }}">{{ adv.name }}</option>
-                        {% endfor %}}
-                    </select>
+        <div class="is-flex is-flex-direction-row mb-4">
+            <div class="is-flex is-flex-direction-column">
+                <label for="layer-selection-adversary" class="pb-2 is-size-6">Generate Layer</label>
+                <div class="is-flex is-flex-direction-row">
+                    <div class="field has-addons">
+                    <div class="control">
+                        <div id="layerSelectionAdversary" class="select">
```
Comment (argaudreau) [inline templates/compass.html]: Can you add `is-small` to the select and button?

File: templates/compass.html [hunk 6]
```diff
@@ -1,127 +1,174 @@
...
+                                <option value="" selected>Select an Adversary (All)</option>
+                                {% for adv in adversaries %}
+                                <option value="{{ adv.adversary_id }}">{{ adv.name }}</option>
+                                {% endfor %}}
+                            </select>
+                        </div>
+                    </div>
+                    <div class="control">
+                    <label for="generateLayer">
+                        <button id="generateLayer" class="button is-primary"
```
Comment (argaudreau) [inline templates/compass.html]: `is-small` is missing here too for this button

**Review by argaudreau — CHANGES_REQUESTED** [review]
Just those two small changes and it's good to merge!

**Review by argaudreau — APPROVED** [review]

**Pull request merged**